### PR TITLE
11 5/add video block

### PR DESCRIPTION
**Pull request by tanvibhakta**
This PR will close ticket https://linear.app/issue/NUWEB-154/[gutenberg]-add-support-for-the-video-block

### CODE REVIEW

**Comment by meetbhatt23** [comment]
@tanvibhakta can you drop links to articles I can use to test this?

**Comment by tanvibhakta** [comment]
https://deploy-preview-44--obvious-wp.netlify.app/articles/droidcon-18-android-path-apis/

**Comment by tanvibhakta** [comment]
This PR will also close tickets https://linear.app/issue/NUWEB-149/[gutenberg]-add-support-for-the-embed-youtube-video-block and https://linear.app/issue/NUWEB-148/[gutenberg]-add-support-for-the-vimeo-block.

Test youtube embeds on these articles:
- https://deploy-preview-44--obvious-wp.netlify.app/articles/simple-a-day-in-the-life
- https://deploy-preview-44--obvious-wp.netlify.app/articles/the-lesser-known-world-of-spans-droidcon-san-francisco

Test vimeo embeds on these articles:
- https://deploy-preview-44--obvious-wp.netlify.app/case-studies/go-jek/
- https://deploy-preview-44--obvious-wp.netlify.app/case-studies/myntra

File: src/styles/embed.css [hunk 1]
```diff
@@ -0,0 +1,5 @@
+.embed-wrapper {
+    /*padding-bottom: calc(var(--aspect-ratio, .5625) * 100%);*/
+    /*aspect-ratio 16:9*/
+    padding-bottom: 56.25%;
+}
```
Comment (meetbhatt23) [inline src/styles/embed.css:5]: We're already using the aspect ratio plugin for tailwind. This shouldn't be necessary. You should use the classes generated from that one. Also as a thumb rule let's have tailwind generate whatever utility css we need. This sort of practice will also ensure we don't have to address the same problem again.

File: src/styles/video.css [hunk 2]
```diff
@@ -0,0 +1,5 @@
+.video-wrapper {
+    /*padding-bottom: calc(var(--aspect-ratio, .5625) * 100%);*/
+    /*aspect-ratio 16:9*/
+    padding-bottom: 56.25%;
+}
```
Comment (meetbhatt23) [inline src/styles/video.css:5]: Same with this one

**Comment by tanvibhakta** [comment]
Vimeo seems to work for me across browsers; Let's do a screenshare during standup today and see what's up?
Youtube should work now - give them a go?

File: src/styles/embed.css (hunk 1 above)
Comment (tanvibhakta) [reply src/styles/embed.css:5]: Wasn't aware of this - I'll work on incorporating this now.

**Comment by meetbhatt23** [comment]
Okay youtube and vimeo work now. Let's fix the aspect ratio thing and I'll get this merged.

**Comment by meetbhatt23** [comment]
Also this branch needs a rebase

**Pull request merged**